package.json

```diff
@@ -33,9 +33,12 @@
     "typescript": "4.9.3"
   },
   "lint-staged": {
-    "**/*": [
+    "**/*.{js,ts}": [
```

Comment:
prettier with `--ignore-unknown` skips unsupported file types so it's safe to cover all files with prettier.

Maybe we can change the strategy to prettier all and eslint js/ts